packages/design-system/src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FunctionComponent, useState } from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash.uniqueid';
+
+interface AccordionItemProps {
+  children?: React.ReactNode;
+  /**
+   * Class to be applied to the heading `<h2>` tag of an accordion item.
+   */
+  contentClassName?: string;
+  /**
+   * Class to be applied to the content `<div>` tag of an accordion item.
+   */
+  headingClassName?: string;
+  /**
+   * Boolean to expand the accordion.
+   */
+  expanded?: boolean;
+  /**
+   * Text for the accordion item heading.
+   */
+  heading: React.ReactNode | string;
+  /**
+   *  If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+}
+
+const AccordionItem: FunctionComponent<AccordionItemProps> = ({
+  children,
+  contentClassName,
```

Comment:
It's strange that github actions shows these `check failures` and yet the PR build shows `All checks have passed`